fbw-a32nx/src/base/flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/CDU/A320_Neo_CDU_IdentPage.js

```diff
@@ -62,11 +64,49 @@ function calculateSecDate(date) {
     } else {
         return "ERR";
     }
+}
+
+function switchDataBase(mcdu)
+    {
+    // Only performing a reset of the MCDU for now, no secondary database
+    //Speed AP returns to selected
+    const isSelected = Simplane.getAutoPilotAirspeedSelected();
+    if (isSelected == false) SimVar.SetSimVarValue("H:A320_Neo_FCU_SPEED_PULL", "boolean", 1);
+    //flight plan
+    mcdu.resetCoroute();
+    mcdu.atsu.atc.resetAtisAutoUpdate();
+    mcdu.flightPlanManager.clearFlightPlan();
+    //stored data
+    mcdu.dataManager.deleteAllStoredWaypoints();
+    //Fuel reset
+    mcdu.zeroFuelWeight = undefined;
+    mcdu.zeroFuelWeightMassCenter = undefined ;
+    mcdu._blockFuelEntered = false;
+    mcdu._fuelPlanningPhase = mcdu._fuelPlanningPhases.PLANNING;
+    mcdu._zeroFuelWeightZFWCGEntered= false;
+    mcdu._blockFuelEntered = false;
+    mcdu.blockFuel = undefined;
+    mcdu.taxiFuelWeight = undefined;
+    //other data
+    mcdu.costIndex = undefined;
+    mcdu.costIndexSet = false;
+    mcdu.flightNumber = undefined
+    mcdu._cruiseEntered =false
+    mcdu._cruiseFlightLevel = undefined;
```

Comment:
Instead of resetting all of the things in here, I suggest using `initVariables` and adding a `resetTakeoffData` param (defaulting to true).

Reply:
@tracernz Hi Tracernz
implementing your suggestion.
I wanted to keep the Threshold reduction/acc and ENG OUT ACC data but for this it would require that I keep also the flight phase at TAKEOFF (FmgcFlightPhases.TAKEOFF) which contradicts the fact we lost the flight plan.
Are you ok if the PERF page looks like this after a DB switch ?
![image](https://user-images.githubusercontent.com/12175224/218273813-ad300a83-a5f6-4ec6-a140-d0b3087a489f.png)
Or do you think we should also keep the flight plan phase too ?

Reply:
Similar issues with FLAP SLAT and Clean Speeds and Trans Alt.
Related to flight plan.